src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -1213,85 +1259,7 @@ def enthalpy_on_nodes(
             phases=phases,
             species=species,
             named_selections=named_selections,
-        )
-
-    def enthalpy_on_faces(
```

Comment:
Why removing this one¿

Reply:
@rafacanton this is an Elemental result from what I have seen. Which would prevent from averaging it onto faces?

Reply:
@PProfizi Because integrated results only make them in their "support". If you have a mass flow rate per face in a surface, you can directly sum all of them and have the overall mass flow rate that flows through the surface (say 20 kg/s). If you average the results to the nodes and then you numerically integrate it along the surface, you cannot guarantee that these 20 kg/s will be conserved. What you would need to do is average the intensive quantity (in this case density * v), average it to the nodes and then integrate it along the surface to get back mass flow rate.

Reply:
@PProfizi Because that is what we were told to do. IMO we should also prevent it in solids, but that is just an opinion (and we have a specific algorithm in Dpf to average integrated results for FEM results, we don't do the same thing than we do for the others).